### ISSUE TITLE
Export to T-SQL

### ISSUE THREAD

**Issue by hristochr** [UziTech/NSF2SQL#6]
Hello and thank you for this tool.
I have been using it for about 2 months now. I am doing what I need to do with the MySQL output files, but still - is it possible / do you plan on integrating export to MSSQL (T-SQL)?

**Comment by UziTech** [comment]
I created this tool when we were switching away from Lotus Notes a few years ago, but haven't touched it since.

Exporting as a SQL file and importing into SQL Server should work.

As far as exporting directly to SQL Server, I have no plans to implement that but would be open to any pull requests.

**Comment by kevingy** [comment]
The SQL exported for MySQL won't work in SQL Server. (The SQL exported for MySQL won't work in MySQL either, without modification, but that's another issue.)

I am actively working on updating the code to work with SQL Server/T-SQL. I have the SQL export file complete. I'm working on direct export to SQL Server now. I'll submit a pull request when complete.

**Comment by kevingy** [comment]
Pull request ready for review.

I don't have a MySQL database to test against. I can't test direct export to a MySQL database.
I don't think that anything was broken, but I can't say for certain that is the case. I was able to confirm that the SQL file exported from the previous code and changed code are the same.

There is one known issue in this pull request. I don't know if this is something that this tool should handle or not. SQL Server has a hard limitation that no single row of data may exceed 8040 bytes. In the sample Domino data that I was using for testing, I exceeded the limit several times. For my testing purposes, I had to comment out the offending lines in the SQL script to get the script to run. I would think at best, NSF2SQL should give the user a warning that the script generated will not run in SQL Server for the reason above and give the offending line numbers so that the user doesn't have to look for them. That has not yet been done. I recommend creating a new issue for it, after this pull request is merged.
